controllers/clustergroupupgrade_controller.go

```diff
@@ -990,16 +1069,22 @@ func (r *ClusterGroupUpgradeReconciler) buildRemediationPlan(ctx context.Context
 			clusters = append(clusters, cluster)
 		}
 	}
-	for i := 0; i < len(clusters); i += clusterGroupUpgrade.Spec.RemediationStrategy.MaxConcurrency {
-		var batch []string
-		for j := i; j < i+clusterGroupUpgrade.Spec.RemediationStrategy.MaxConcurrency && j != len(clusters); j++ {
-			site := clusters[j]
-			if !isCanary[site] {
-				batch = append(batch, site)
-			}
+
+	var batch []string
+	batchCount := 0
+	for i := 0; i < len(clusters); i++ {
+		site := clusters[i]
+		if !isCanary[site] && clusterNonCompliantWithManagedPoliciesMap[site] == true {
+			batch = append(batch, site)
```

Comment:
When there is a list of policies to be remediated does the operator remediate the policies in the order they are listed?

Reply:
It will need to remediate in the order they are listed in the CR and wait until the previous one is remediate before continuing.

Reply:
Yes. I have a separate ticket to update the reconcile logic for doing that.

Reply:
I do remember we said that we will assign a priority/order value to the policy ? default is 0 for all if there is priority/order so it needs to be defined. May be future enhancement ?